README.md

```diff
@@ -1 +1,133 @@
-# CR-portfolio-project
\ No newline at end of file
+<a name="readme-top"></a>
+
+<div align="center">
+  <img src="media/png/screenshot.png" alt="logo" width="140"  height="auto" />
+  <br/>
+
+  <h3><b>Portfolio</b></h3>
+
+</div>
+
+<!-- TABLE OF CONTENTS -->
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Usage](#usage)
+- [👥 Authors](#authors)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 [CR Portfolio Project] <a name="about-project"></a>
+
+Code review portfolio project. Build bor the code review code training.
+
+**[your_project__name]** is a...
```

Comment:
Good job implementing the correct template 👍 . However, there are some issues that need to be fixed in order to meet the requirements:

- [ ] Kindly delete any text from the template that is a placeholder, like "**[your_project__name]**". This would help your README look more professional.

- [ ] Please, add a Future Features section to your template. This will show other people that you are determined in improving your app 💯 .